packages/frontend/src/features/semanticViewer/runners/SemanticViewerResultsRunner.ts

```diff
@@ -64,30 +65,52 @@ function getVizIndexTypeFromDimensionType(
     }
 }
 
-export class SemanticViewerResultsRunner extends ResultsRunner {
+export class SemanticViewerResultsRunner
+    implements IResultsRunner<VizChartLayout>
```

Comment:
Here `VizChartLayout` should be `SqlRunnerPivotLayout` and it can be known to this object.